src/publisher/api_v2_views.py

```diff
@@ -14,10 +15,55 @@
 from et3.render import render_item
 import logging
 from django.http.multipartparser import parse_header
+import django.db.transaction
 
 
 LOG = logging.getLogger(__name__)
 
+PROFILING = False
+
+
+def profile(fn):
```

Comment:
there is some overlap of this decorator with one in logic.

add a note, I created this one believing something like it didn't exist :(